src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/IndexViewModel.kt

```diff
@@ -75,7 +75,11 @@ data class IndexViewModel(@Serialise("reportsJson") val reports: List<ReportRowV
             val canConfigure = context.hasPermission(ReifiedPermission("reports.configure", Scope.Global()))
             val reportDisplayNames = if (canConfigure)
             {
-                reportRows.filter{ it.ttParent == 0 }.map{ it.name to it.displayName }.toMap()
+                val reportsWithPublished = reportRows.filter{ it.ttParent != 0 && it.published == true }
```

Comment:
Because we've defined pinned report as always the latest published version, avoid confusion by only making reports with at least one published version available to the typeahead. Otherwise, could add a report as pinned but then it wouldn't show up.